### PR TITLE
@l2succes: track social authentication analytics properties

### PR DESCRIPTION
**Pull request by xtina-starr**
Dependent on https://github.com/artsy/reaction/pull/964

This is the second part of https://artsyproduct.atlassian.net/browse/GROW-780 that ensures we are tracking the 'created account' and 'successfully logged in' events.

### CODE REVIEW

**Comment by xtina-starr** [comment]
The test are failing for this but I believe that's because reaction needs to be bumped up when [this PR](https://github.com/artsy/reaction/pull/964) gets merged

**Comment by l2succes** [comment]
@xtina-starr Looks good, let's bump reaction on this PR to get it green

File: src/desktop/apps/auth2/components/ModalContainer.tsx [hunk 1]
```diff
@@ -39,6 +42,30 @@ export const ModalContainer: React.SFC<any> = () => {
...
+          modal_copy: data.copy,
+          trigger: data.trigger || 'click',
+          trigger_seconds: data.triggerSeconds,
+          intent: data.intent,
+          auth_redirect: data.redirectTo || data.destination,
+          service: data.service,
+        }
+
+        Cookies.set(
+          `analytics-${data.mode}`,
```
Comment (xtina-starr) [inline src/desktop/apps/auth2/components/ModalContainer.tsx]: The major change is here. I decided two use two distinct cookies for analytics tracking `analytics-signup` and `analytics-login`. I think this makes think a little more clear with the separation.

File: src/desktop/analytics/account_login.js [hunk 2]
```diff
@@ -7,6 +7,17 @@ var trackViewLogin = function() {
   analytics.track('Viewed login form')
 }
 
+const trackLogin = options => {
+  analytics.track('Successfully logged in', options)
```
Comment (xtina-starr) [inline src/desktop/analytics/account_login.js]: I added a separate tracking event for login here.

Comment (xtina-starr) [reply src/desktop/analytics/account_login.js]: Before we were tracking successful login event via the `auth:login` mediator. It was located within the `account_creation.js` file which was a little confusing. I'm thinking we can deprecate `auth:login` once we flip the switch for the new auth.

File: src/desktop/apps/auth2/components/ModalContainer.tsx (hunk 1 above)
Comment (l2succes) [reply src/desktop/apps/auth2/components/ModalContainer.tsx]: Cool that's fine, but that also means we have to be more careful with clearing the right cookies after using them in the legacy code right?

Comment (xtina-starr) [reply src/desktop/apps/auth2/components/ModalContainer.tsx]: Yeah, that is in part why I separated them. I wanted to ensure we cleared out the correct one. When they weren't separated I had to use a bunch of conditional logic to ensure I was expiring the right one.

Comment (xtina-starr) [reply src/desktop/apps/auth2/components/ModalContainer.tsx]: I can walk you through it IRL.

**Pull request merged**